### PR TITLE
update TS to 3.5

### PR DESCRIPTION
**Pull request by theKashey**
+ bump react and other packages. It's time.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,19 +37,19 @@
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.4.0",
     "gh-pages": "^1.1.0",
-    "jest": "^23.5.0",
+    "jest": "^24.8.0",
     "prettier": "^1.11.1",
     "prettier-eslint-cli": "^4.7.1",
-    "react": "^16.3.0",
-    "react-dom": "^16.2.0",
+    "react": "^16.8.6",
```
Comment (zzarcon) [inline package.json:44]: we should update the `peerDependencies` from `src/commands/init.ts` then

File: package.json [hunk 2]
```diff
@@ -37,19 +37,19 @@
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.4.0",
     "gh-pages": "^1.1.0",
-    "jest": "^23.5.0",
+    "jest": "^24.8.0",
```
Comment (zzarcon) [inline package.json:41]: Do we need to make any changes in the `test` script? If we just had tests... 😸

File: package.json [hunk 3]
```diff
@@ -21,8 +21,8 @@
   },
   "dependencies": {
     "@types/enzyme": "^3.1.9",
-    "@types/jest": "^23.3.1",
-    "@types/react": "^16.0.40",
+    "@types/jest": "^24.0.15",
+    "@types/react": "^16.8.22",
     "@types/react-dom": "^16.0.4",
```
Comment (zzarcon) [inline package.json]: Bump this as well?

File: package.json [hunk 4]
```diff
@@ -37,19 +37,19 @@
...
+    "react-dom": "^16.8.6",
     "react-gh-corner": "^1.1.2",
     "styled-components": "^3.3.2",
     "tmp": "^0.0.33",
     "tslib": "^1.9.3",
     "tslint": "^5.9.1",
     "tslint-config-prettier": "^1.10.0",
     "tslint-react": "^3.5.1",
-    "typescript": "^2.7.2",
+    "typescript": "^3.5.2",
```
Comment (zzarcon) [inline package.json:53]: 🙏

**Comment by theKashey** [comment]
🎉

**Comment by zzarcon** [comment]
🎈

**Comment by theKashey** [comment]
(does not count unless released)

**Pull request merged**